spec/support/matchers/match_collection.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+RSpec::Matchers.define :match_collection do |expected_collection, attribute_names: []|
```

Comment:
Whats going on here? This isn't referenced in the code

Reply:
It is. More above: https://github.com/DFE-Digital/publish-teacher-training/pull/4751/files#diff-feadbcd7cd23dccc9c8311aea6c91bca39ac979ae097ec092b74e48b1168ec75R38